src/test_typing_extensions.py

```diff
@@ -1978,6 +1986,13 @@ class PointDict3D(PointDict2D, total=False):
         assert is_typeddict(PointDict2D) is True
         assert is_typeddict(PointDict3D) is True
 
+    @skipUnless(HAS_FORWARD_MODULE, "ForwardRef.__forward_module__ was added in 3.9")
+    def test_get_type_hints_cross_module_subclass(self):
+        self.assertEqual(
```

Comment:
Drive-by suggestion: you could maybe add an assertion here to verify that the test is working as intended, and `_DoNotImport` has not been imported 

```suggestion
        # Test the test: _DoNotImport cannot be in globals() or locals(),
        # or the second assertion will pass for the wrong reasons
        self.assertNotIn("_DoNotImport", {**globals(), **locals()})
        self.assertEqual(
```

Reply:
Done!